Libraries/acdriver/Sources/Options.cpp

```diff
@@ -84,6 +84,10 @@ parseArgument(std::vector<std::string> const &args, std::vector<std::string>::co
         return libutil::Options::Next<std::string>(&_filterForDeviceOsVersion, args, it);
     } else if (arg == "--asset-pack-output-specifications") {
         return libutil::Options::Next<std::string>(&_assetPackOutputSpecifications, args, it);
+    } else if (arg == "--allow-extensions") {
```

Comment:
I'd honestly move this into their own function or the top of the parsing logic -- something to visually distinguish them as extensions.  Comment as well?